modules/ROOT/pages/migration/index.adoc

```diff
@@ -16,14 +16,389 @@ To update your Neo4j GraphQL Library, use npm or the package manager of choice:
+Here an examples of some of the new filters available:
+
+```graphql
+input StringWhere {
+  AND: [StringWhere!]
+  NOT: StringWhere
+  OR: [StringWhere!]
+  contains: String
+  endsWith: String
+  equals: String
+  in: [String!]
+  startsWith: String
+}
+
+input BooleanWhere {
+  AND: [BooleanWhere!]
+  NOT: BooleanWhere
+  OR: [BooleanWhere!]
+  equals: Boolean
+}
+
+input FloatWhere {
+  AND: [FloatWhere!]
+  NOT: FloatWhere
+  OR: [FloatWhere!]
+  equals: Float
+  gt: Float
+  gte: Float
+  in: [Float!]
+  lt: Float
+  lte: Float
+}
+```
+
+*Property filters*
+
+In version 5, you could filter by a field like this:
+
+```graphql
+query {
+  moviesConnection(where: { title_CONTAINS: "The Matrix" }) {
+    edges {
+      node {
+        title
+      }
+    }
+  }
+}
+```
+
+With the new filters in version 6, you can filter like this:
+
+```graphql
+query {
+  movies(
+    where: {
+      node: {
+        title: { contains: "The Matrix" }
+      }
+    }
+  ) {
+    connection {
+      edges {
+        node {
+          title
+        }
+      }
+    }
+  }
+}
+```
+
+Note that in version 5, the equality operator came without any suffixes, and to filter `title` you would have done by the filter `{ title: "The Matrix" }` while in version 6 the filter is moved in the nested input under the operator `equals` as: `{ title: { equals: "The Matrix" } }`.
+
+*Relationship filters*
+
+The same approach is also embraced by relationship filters.
+
+Filter all the movies that have at least one actor with the name "Keanu Reeves":
+
+Version 5:
+
+```graphql
+{
+  moviesConnection(
+    where: { actorsConnection_SOME: { node: { name: "Keanu Reeves" } } }
+  ) {
+    edges {
+      node {
+        title
+      }
+    }
+  }
+}
+```
+
+Version 6: 
+
+```graphql
+query {
+  movies(
+    where: {
+      node: {
+        actors: { edges: { some: { node: { name: { equals: "Keanu Reeves" } } } } }
+      }
+    }
+  ) {
+    connection {
+      edges {
+        node {
+          title
+        }
+      }
+    }
+  }
+}
+```
+
+Note how in version 6, the filter `some` is not a suffix anymore but a nested field under the `edges` field.
+
+*Relationship properties filters*
+
+Filter all the movies where some actors have a screen time greater than 100 minutes:
+
+Version 5:
+
+```graphql
+{
+  moviesConnection(
+    where: { actorsConnection_SOME: { edge: { screenTime_GT: 10 } } }
+  ) {
+    edges {
+      node {
+        title
+      }
+    }
+  }
+}
+```
+Version 6:
+
+```graphql
+query {
+  movies(
+    where: {
+      node: {
+        actors: {
+          edges: { some: { properties: { screenTime: { gt: 1999 } } } }
+        }
+      }
+    }
+  ) {
+    connection {
+      edges {
+        node {
+          title
+        }
+      }
+    }
+  }
+}
+```
```

Comment:
Why are we not using the tabs here? I see we are using them afterwards, and seems like a good idea here